### PR TITLE
feat: Layout component

### LINKED ISSUE #216: Create Layout component with left or right column
Instead of writing multiple time the same classes for the layout we should create a component to manage this. It could work like this:
```
<Layout>
  <template slot="sidebar-left">Sidebar content</template>
  <template slot="content-right">Content</template>
</Layout>
```
or
```
<Layout>
  <template slot="sidebar-right">Sidebar content</template>
  <template slot="content-left">Content</template>
</Layout>
```
(PS the left and right sidebar aren't the same size)

This would replace what use in all the pages like this:
```
<Container :slim="true">
  <div>
    <div class="col-12 col-lg-3 float-left hide-sm hide-md hide-lg">
      Sidebar content
    </div>
    <div class="col-12 col-lg-9 float-left pl-0 pl-lg-5">
      Content
    </div>
  </div>
</Container>
  ```

### PR DESCRIPTION
**Pull request by samuveth**
Fixes #216 

Changes proposed in this pull request:
- Move content/sidebar layout classes to a layout component 
- 
-

### CODE REVIEW

File: src/views/Strategy.vue [hunk 1]
```diff
@@ -6,49 +6,50 @@
       </router-link>
     </div>
     <div>
-      <div class="col-12 col-lg-8 float-left pr-0 pr-lg-5">
```
Comment (bonustrack) [inline src/views/Strategy.vue:9]: The strategy page is broken the content is not shown https://bafybeie7ui2lll252cv3tekjocx7zs2x7wuvujyiudpnwdsviutpgsrajm.on.fleek.co/#/strategy/erc20-balance-of-delegation

File: src/components/Layout.vue [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<template>
+  <div>
+    <div class="col-12 col-lg-3 float-left hide-sm hide-md hide-lg">
```
Comment (bonustrack) [inline src/components/Layout.vue:3]: This should not appear if the slot doesn't exist on Vue2 you can do like this
```suggestion
    <div v-if="$slots['sidebar-left']" class="col-12 col-lg-3 float-left hide-sm hide-md hide-lg">
```

File: src/components/Layout.vue [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<template>
+  <div>
```
Comment (bonustrack) [inline src/components/Layout.vue:2]: - The component should include this
```<Container :slim="true">```
(And Container should be removed from the views if using Layout)
- The breadcrumb back / home / space link can be moved inside the Layout in the "content-left" or "content-right" slot

**Comment by bonustrack** [comment]
@samuveth Hum, the back button is missing in all the pages like Proposal / Settings / Setup etc. Maybe there been some misunderstanding but it should not be removed.

**Comment by samuveth** [comment]
No, I just miss places it in the template 😉

**Review by bonustrack — APPROVED** [review]

**Pull request merged**